CHANGELOG.md

```diff
@@ -4,12 +4,12 @@
 
 ### Added
 
-- Added MMDS metrics for `GET`, `PUT` and `PATCH` requests
 - Added devtool build `--ssh-keys` flag to support fetching from private
   git repositories.
 - Added option to configure block device flush.
 - Added `--new-pid-ns` flag to the Jailer in order to spawn the Firecracker
   process in a new PID namespace.
+- Added MMDS metrics for `GET`, `PUT` and `PATCH` requests
```

Comment:
3rd commit is a _fix_ for 2nd commit, like I said in earlier comment, just squash all commits together.

Reply:
Done